topdown/http.go

```diff
@@ -95,6 +97,8 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 		} else if key == "url" {
```

Comment:
🤔 So many `if/else`, this could also read nicely as a `switch` statement, I guess 😉